### DISCUSSION TITLE
[Suggestion] Add Death Knights with TBC release

### DISCUSSION THREAD

**Discussion by Santaria94** [chromiecraft/chromiecraft#4533] [Suggestions]
Suggestion to add Death Knights with TBC release. Reasons for adding this class:

1. Finding potential bugs in current content related to the Death Knight. As an example for Nefarian, a new class call has been added with the release of Wotlk. Does it work now? We do not know. Why? Because we can't play DK.
There may also be various bugs associated with the abilities of the DK. One such ability is Death Grip, as far as I know on many private projects, many mobs and even bosses that should be immune to it do not have this immunity. Perhaps there are similar problems on Azerothcore.
2. Death Knight is a minimum of 1 additional alt for all active players, which quickly reaches the level cap. This can help with raid recruitment and also potentially increase the number of raids. It would also increase the number of tanks in the RDF, which are usually in short supply.
3. As far as I remember, in discussions about death knights in brackets 50-59 and 60, it was said that this class is unbalanced due to its starting equipment. But with the release of TBC and Brackets 61-64, all classes will get readily available gear from TBC quests and dungeons that outperforms Death Knight starting gear.
4. Also I think it will be just fun to try classic and TBK content with a class that wasn't there originally. That would be a cool feature of the server.

In general, both in terms of testing and game moments, I see no reason not to add this class.
And if one of the main goals of the project is to fully test the content of 3.3.5, then it would be a big oversight not to add this class now, since many bugs associated with this class in Classic / TBC raids and dungeons may go unnoticed

**Comment by 55Honey** [comment]
Hello Santaria94,
Thanks for your suggestion.
I'll make it short: no. This won't happen.
It was discussed and though over. It's not blizzlike and not doable at all.
